components/webcompat_reporter/ui/webcompat_reporter.tsx

```diff
@@ -0,0 +1,70 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { render } from 'react-dom'
+import { Provider } from 'react-redux'
+import { bindActionCreators } from 'redux'
+
+import Theme from 'brave-ui/theme/brave-default'
+import DarkTheme from 'brave-ui/theme/brave-dark'
+import BraveCoreThemeProvider from '../../common/BraveCoreThemeProvider'
+require('../../fonts/muli.css')
+require('../../fonts/poppins.css')
```

Comment:
These files were moved on master, unfortunately. You'll find them somewhere like `brave/ui/webui/fonts`